build error: OSCNode.cpp:153:28: error: cannot convert 'lo_arg*' to 'lo_blob' {aka 'lo_blob_*'} in initialization

I get 

```
plugins/osc/OSCNode.cpp: In function 'int ola::plugin::osc::OSCDataHandler(const char*, const char*, lo_arg**, int, void*, void*)':
plugins/osc/OSCNode.cpp:153:28: error: cannot convert 'lo_arg*' to 'lo_blob' {aka 'lo_blob_*'} in initialization
  153 |       lo_blob blob = argv[0];
      |                      ~~~~~~^
      |                            |
      |                            lo_arg*
plugins/osc/OSCNode.cpp: In member function 'bool ola::plugin::osc::OSCNode::Init()':
plugins/osc/OSCNode.cpp:266:50: error: invalid conversion from 'int (*)(const char*, const char*, lo_arg**, int, void*, void*)' to 'lo_method_handler' {aka 'int (*)(const char*, const char*, lo_arg**, int, lo_message_*, void*)'} [-fpermissive]
  266 |   lo_server_add_method(m_osc_server, NULL, NULL, OSCDataHandler, this);
      |                                                  ^~~~~~~~~~~~~~
      |                                                  |
      |                                                  int (*)(const char*, const char*, lo_arg**, int, void*, void*)
In file included from /usr/include/lo/lo.h:32,
                 from ./plugins/osc/OSCNode.h:24:
/usr/include/lo/lo_lowlevel.h:876:72: note:   initializing argument 4 of 'lo_method_* lo_server_add_method(lo_server, const char*, const char*, lo_method_handler, const void*)'
  876 |                                const char *typespec, lo_method_handler h,
      |                                                      ~~~~~~~~~~~~~~~~~~^
make[2]: *** [Makefile:12250: plugins/osc/libolaoscnode_la-OSCNode.lo] Error 1
make[2]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make[1]: *** [Makefile:16480: all-recursive] Error 1
make[1]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make: *** [Makefile:7553: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...

```

during the build.
Is someone else having this, too?

--- a/plugins/osc/OSCNode.cpp
+++ b/plugins/osc/OSCNode.cpp
@@ -141,7 +141,7 @@ bool ExtractSlotValueFromPair(const string &type, lo_arg **argv,
  * @param user_data a pointer to the OSCNode object.
  */
 int OSCDataHandler(const char *osc_address, const char *types, lo_arg **argv,
-                   int argc, void *, void *user_data) {
+                   int argc, lo_message, void *user_data) {
   OLA_DEBUG << "Got OSC message for " << osc_address << ", types are " << types;
 
   OSCNode *node = reinterpret_cast<OSCNode*>(user_data);
@@ -150,7 +150,7 @@ int OSCDataHandler(const char *osc_address, const char *types, lo_arg **argv,
 
   if (argc == 1) {
     if (type == "b") {
-      lo_blob blob = argv[0];
+      lo_blob blob = (lo_blob)argv[0];
       unsigned int size = min(static_cast<uint32_t>(DMX_UNIVERSE_SIZE),
                               lo_blob_datasize(blob));
       node->SetUniverse(

